sublinks-ui/services.yml

```diff
@@ -0,0 +1,32 @@
+version: "3.7"
+
+x-logging: &default-logging
+  driver: "json-file"
+  options:
+    max-size: "50m"
+    max-file: "4"
+
+include:
+  - ../common/proxy.yml
+
+services:
+  ui:
+    image: ghcr.io/sublinks/sublinks-frontend:main
+    environment:
+      - NEXT_PUBLIC_SUBLINKS_API_BASE_URL=sublinks:8080
+      - NEXT_PUBLIC_SUBLINKS_API_BASE_PUBLIC_URL=localhost
+      - NEXT_PUBLIC_HTTPS_ENABLED=false
```

Comment:
@sublinks/sublinks-frontend-service need someone to confirm if this is the correct expected environment variables when running the UI in "production" mode